src/qml/controls/Information.qml

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) 2022 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.15
+
+Control {
```

Comment:
As the description of a17b84ec675c8ab27a560d9d301c8f8d96d90443 states, "This is an object similar to Setting.qml". I think that instead of having this control we can just refactor `Setting.qml` to load the desired component in place of the currently hardcoded switch. This adds flexibility to the setting control to fit in with all the use cases required of it within options handling without introducing a new component here, and other components when a new component is desired in the switch's position.

Reply:
This would be the ideal way to move forward, and this was something I tried to do while beginning to write this PR.

However, the crucial problem I faced while going with this idea at the beginning was that QML is a declarative language and hence doesn't support conditional statements that are more complicated than a ternary operator.
So it was quite hard to figure out a way to make `Settings.qml` switch the type of _description_ based on the option type without using conditional statements. So I went forward by creating `Information.qml` from scratch and, one by one, incorporated changes to work with all the option types. 

But now, since the `Information.qml` has been laid out, and almost all the permutations of various options have been accounted for, we can think of ways to merge these two files effectively.